source/common/network/connection_impl.cc

```diff
@@ -753,6 +756,27 @@ void ConnectionImpl::flushWriteBuffer() {
   }
 }
 
+std::ostream& operator<<(std::ostream& os, Connection::State connection_state) {
+  switch (connection_state) {
+  case Connection::State::Open:
+    return os << "Open";
+  case Connection::State::Closing:
+    return os << "Closing";
+  case Connection::State::Closed:
+    return os << "Closed";
+  }
+  return os;
+}
+
+// ScopeTrackedObject
```

Comment:
Please remove comment. We don't seem to follow the practice of tagging interface methods in cc files.